src/sources/file.rs

```diff
@@ -70,11 +68,13 @@ enum BuildError {
 }
 
 /// Configuration for the `file` source.
+#[serde_as]
```

Comment:
I'm not seeing anything that is using `serde_as` here.

Reply:
Ah, old and didn't get removed. Was trying to replace the `Option<u64>`'s that are really Durations but it doesn't generate correctly yet.

Reply:
Resolved in [192c429](https://github.com/vectordotdev/vector/pull/16099/commits/192c429cac97c0da46a361108a4d292eeef97f2a)